src/mbgl/style/layer_impl.hpp

```diff
@@ -42,6 +36,7 @@ class Layer::Impl {
 
     // Create a new layer with the specified `id` and `sourceID`. All other properties
     // are copied from this layer.
+    // TODO: evaluate if this still meets expectations
```

Comment:
`copy` is only used in one place: `StyleSourcedAnnotationImpl`, which is code supporting a pre-runtime styling feature that we should remove. We can save that for a separate change though.